src/accessors-geog.cpp

```diff
@@ -63,70 +47,66 @@ double perimeter(PyObjectGeography a, double radius = numeric_constants::EARTH_R
 }
 
 void init_accessors(py::module& m) {
+    py::options options;
+    options.disable_function_signatures();
+
     m.attr("EARTH_RADIUS_METERS") = py::float_(numeric_constants::EARTH_RADIUS_METERS);
 
     m.def("centroid",
           py::vectorize(&centroid),
-          py::arg("a"),
-          R"pbdoc(
+          py::arg("geography"),
```

Comment:
While we are at it (renaming those single args), should we make those first arguments positional-only? (so we avoid anyone ever doing `centroid(geography=...)` (renaming this keyword name is really only meant for being descriptive in the docs, I would say?)